.github/workflows/specs.yml

```
@@ -0,0 +1,22 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: [2.7.7, 3.1.3]
```

```suggestion
        ruby-version: [2.7, 3.1]
```
This way it's easier to stay on the latest versions.